apis/v1/resourceSnapshot_types.go

```diff
@@ -11,6 +11,15 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+const (
+	// ResourceIndexLabel is the label that indicate the resource snapshot index of a cluster policy.
+	ResourceIndexLabel = labelPrefix + "policyIndex"
```

Comment:
does this label store the index of the resourceSnapshot for the pagination, or policySnapshot info?

i'm confused by the comment and the label value "policyIndex".

Reply:
```suggestion
	ResourceIndexLabel = labelPrefix + "resourceIndex"
```
missed the comment?